packages/maskbook/src/plugins/ITO/schema-v1.json

```diff
@@ -119,19 +119,6 @@
         "seller": {
             "$ref": "#/definitions/trader"
         },
-        "buyers": {
```

Comment:
Remove the useless `buyers` property. Through test, it would not break the old payload since `additionalProperties` is `true`, and `buyers` is also removed form `required` list.